nupic/regions/TPRegion.py

```diff
@@ -209,6 +210,14 @@ def getConstraints(arg):
       count=1,
       constraints='bool'),
 
+    unionMode=dict(
```

Comment:
I wouldn't call this `unionMode` since the requirement is not specific to union pooling. I would call it something more descriptive and accurate like `computePredictedActiveCellIndices`.